runelite-client/src/main/java/net/runelite/client/Notifier.java

```diff
@@ -169,9 +180,30 @@ else if (client.getGameState() != GameState.LOGGED_IN)
 		graphics.fill(new Rectangle(client.getCanvas().getSize()));
 		graphics.setColor(color);
 
-		if (Instant.now().minusMillis(FLASH_DURATION).isAfter(flashStart))
+		switch (notificationFlashType)
 		{
-			flashStart = null;
+			case FLASH_TWO_SECONDS:
+			case SOLID_TWO_SECONDS:
+				if (Instant.now().minusMillis(2000).isAfter(flashStart))
```

Comment:
```suggestion
				if (Instant.now().minusMillis(MINIMUM_FLASH_DURATION_BEFORE_CANCELING_MILLIS).isAfter(flashStart))
```

Reply:
Also, this check is used in all branches of this switch statement. Is there any reason it's not a guard clause before the switch starts?

Reply:
The config options are "Flash/Solid for two seconds" so I figured I probably shouldn't use the minimum duration and assume it will be 2 seconds because that constant could be updated.

That being said, I don't think there's much reason to ever change that so perhaps I could just call the constant `TWO_SECOND_MINIMUM_FLASH_DURATION_MILLIS` and make a guard clause out of it?

Reply:
I think it's most likely that if it changes, it would change to become some value we look up from user config, so it's probably fine to reference the constant there.